src/core/Akka/Util/ThreadLocalRandom.cs

```diff
@@ -28,7 +29,13 @@ public static Random Current
         {
             get
             {
-                return _rng.Value;
+                if (_rng == null)
```

Comment:
Why this change?

Reply:
Paranoia change, but Reason is documented here:

https://ayende.com/blog/189761-A/production-postmortem-the-slow-slowdown-of-large-systems